tests/test_enhanced_error_classification_simplified.py

```
@@ -0,0 +1,355 @@
+            line=5, column=80,
+            rule_id="E501",
+            message="line too long (85 > 79 characters)",
+            severity=ErrorSeverity.WARNING,
+            linter="flake8"
+        )
+        
+        category = self.error_analyzer._categorize_error(style_warning)
+        priority = self.error_analyzer._calculate_priority(style_warning, category)
+        
+        # Style warnings should have lower priority
+        assert priority <= 50
```

Ruff (0.12.2)

Use of `assert` detected

(S101)